internal/activity.go

```diff
@@ -60,6 +60,11 @@ type (
 		IsLocalActivity   bool          // true if it is a local activity
 	}
 
+	// ActivityExtraInformation contains additional data to be sent to the server on activity failure
+	ActivityExtraInformation struct {
+		retryDelay *time.Duration
```

Comment:
```suggestion
		RetryDelay time.Duration
```

Can we just make this a public field like we do with activity info? Also, would prefer if we could avoid pointers to primitives like this and instead just assume unset/0 means unset.